src/app/utils/BountyUtils.ts

```diff
@@ -262,6 +269,206 @@ const BountyUtils = {
+                    footer = { text: '📮 - submit | ✅ - mark complete | 💰 - mark paid | 🆘 - help', };
+                    reacts.push('💰');
+                } else {
+                    footer = { text: '📮 - submit | ✅ - mark complete | 🆘 - help', };
+                }
+                reacts.push('🆘');
+                fields.push({ name: 'Claimed by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.claimedBy.discordId, bounty.customerId)).user.tag, inline: true });
+                if (bounty.paidStatus === PaidStatus.paid) fields.push({ name: 'Paid by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.createdBy.discordId, bounty.customerId)).user.tag, inline: true });
+                break;
+            case BountyStatus.in_review:
+                color = '#d39e00';
+                reacts.push('✅');
+                if (bounty.paidStatus !== PaidStatus.paid) {
+                    footer = { text: '✅ - mark complete | 💰 - mark paid | 🆘 - help', };
+                    reacts.push('💰');
+                } else {
+                    footer = { text: '✅ - mark complete | 🆘 - help', };
+                }
+                reacts.push('🆘');
+                fields.push({ name: 'Claimed by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.claimedBy.discordId, bounty.customerId)).user.tag, inline: true });
+                fields.push({ name: 'Submitted by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.submittedBy.discordId, bounty.customerId)).user.tag, inline: true });
+                if (bounty.paidStatus === PaidStatus.paid) fields.push({ name: 'Paid by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.createdBy.discordId, bounty.customerId)).user.tag, inline: true });
+                break;
+            case BountyStatus.complete:
+                color = '#01d212';
+                reacts.push('🔥');
+                if (bounty.paidStatus !== PaidStatus.paid) {
+                    footer = { text: '💰 - mark paid', };
+                    reacts.push('💰');
+                }
+                fields.push({ name: 'Claimed by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.claimedBy.discordId, bounty.customerId)).user.tag, inline: true });
+                // Bounty might jump directly to Complete status so these would be null...
+                if (!!bounty.submittedBy) fields.push({ name: 'Submitted by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.submittedBy.discordId, bounty.customerId)).user.tag, inline: true });
+                if (!!bounty.reviewedBy) fields.push({ name: 'Reviewed by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.reviewedBy.discordId, bounty.customerId)).user.tag, inline: true });
+                if (bounty.paidStatus === PaidStatus.paid) fields.push({ name: 'Paid by', value: (await DiscordUtils.getGuildMemberFromUserId(bounty.createdBy.discordId, bounty.customerId)).user.tag, inline: true });
```

Comment:
nit: I think you can extract this logic outside of the status based conditional flow. Our validation logic will handle applying a paid status to the correct lifecycle status of a bounty